sources/api/certdog/src/main.rs

```diff
@@ -54,30 +57,29 @@ struct CertBundle {
 
 /// Query the API for the certificate bundles, returns a tuple with trusted
 /// and distrusted PEM certificates
-async fn get_certificate_bundles<P>(socket_path: P) -> Result<CertBundle>
+async fn get_certificate_bundles<P>(config_path: P) -> Result<CertBundle>
 where
     P: AsRef<Path>,
 {
-    debug!("Querying the API for settings");
-
-    let method = "GET";
-    let uri = constants::API_SETTINGS_URI;
-    let (_code, response_body) = apiclient::raw_request(&socket_path, uri, method, None)
-        .await
-        .context(error::APIRequestSnafu { method, uri })?;
-
-    // Build a Settings struct from the response string
-    debug!("Deserializing response");
-    let settings: model::Settings =
-        serde_json::from_str(&response_body).context(error::ResponseJsonSnafu { uri })?;
-
-    split_bundles(settings.pki.unwrap_or_default())
+    debug!("Loading certdog configuration");
+    if config_path.as_ref().exists() {
+        let config_str =
+            fs::read_to_string(config_path.as_ref()).context(error::ConfigReadSnafu {
+                config: config_path.as_ref().to_str().unwrap(),
+            })?;
+        let pki: HashMap<Identifier, PemCertificate> = toml::from_str(config_str.as_str())
+            .context(error::ConfigDeserializationSnafu {
+                config: config_path.as_ref().to_str().unwrap(),
```

Comment:
If what you want is the path string, you can try:

```suggestion
                config: format!("{:?}", config_path),
```

With:

```rust
println!("{:?}", &Path::new("My").join("Cool").join("Path"));
```

You will see:

```
"My/Cool/Path"
```